boards/posix/native_posix/doc/board.rst

```diff
@@ -386,23 +460,46 @@ The following peripherals are currently provided with this board:
     running :ref:`Shell`. For more information refer to the section
     `Shell support`_.
 
-**Simple timer**:
-  A simple timer provides the kernel with a 10ms tick.
+**Clock, timer and system tick model**
+  This model provides the system tick timer. By default
+  :option:`CONFIG_SYS_CLOCK_TICKS_PER_SEC` configures it to tick every 10ms.
```

Comment:
note that use of single back ticks is correct when it's used for ``:ref:`name` `` or ``:option:`name` `` and other roles.